src/execution/operator/persistent/physical_export.cpp

```diff
@@ -46,7 +47,15 @@ static void WriteCopyStatement(FileSystem &fs, stringstream &ss, TableCatalogEnt
 	} else {
 		table_file_path = fs.JoinPath(info.file_path, StringUtil::Format("%s.%s", table->name, function.extension));
 	}
-	ss << table->name << " FROM '" << table_file_path << "' (";
+
+	if(KeywordHelper::RequiresQuotes(table->name)){
+		ss << '"'  << table->name << '"' ;
```

Comment:
Perhaps use `KeywordHelper::WriteOptionallyQuoted`? That would simplify this a bit.